package.json

```diff
@@ -16,7 +16,7 @@
     "test:dev:verbose": "jest --watch \"^((?!slow).)*$\"",
     "build": "tsc && cp -R ./src/code-templates/ ./.dist/src/code-templates",
     "build:watch": "tsc --watch",
-    "start:dev:reload": "nodemon ./src/code-generator/index.js"
+    "start:dev:reload": "nodemon ./bin/cli.ts"
```

Comment:
On build we don't build the `bin/cli.ts` file, is this the intended behavior?

Reply:
@rluvaton Why we don't build it? It is not excluded anywhere or what do I miss?